src/components/pages/ParentFaq/ParentFaqContainer.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { Layout, Card, Row, Col, Button } from 'antd';
+import img1 from './img1.png';
+import ParentNavTopBar from '../../common/ParentNavTopBar';
+
+const ParentFaqContainer = () => {
+  return (
+    <>
+      <Layout className="parent-dashboard">
+        <ParentNavTopBar />
+        <div className='"site-card-border-less-wrapper"'>
+          <Row className="card">
+            <Col span={16} className="col-btn">
+              <Row>
+                <h4 style={{ color: '#878D93', fontWeight: '700' }}>
+                  FREQUENTLY ASKED QUESTIONS
+                </h4>
+              </Row>
+              <div className="btn-group">
+                <Row>
+                  <Button type="default" style={{ backgroundColor: '#fbfbfb' }}>
+                    Game Play
+                  </Button>
+                  <Button type="default" style={{ backgroundColor: '#fbfbfb' }}>
+                    Child Profile Management
+                  </Button>
+                </Row>
+                <Row>
+                  <Button type="default" style={{ backgroundColor: '#fbfbfb' }}>
+                    Account Setting
+                  </Button>
+                  <Button type="default" style={{ backgroundColor: '#fbfbfb' }}>
+                    Technical Support
+                  </Button>
+                </Row>
+              </div>
+            </Col>
+            <Col
+              span={8}
+              type="flex"
+              style={{ alignItems: 'center' }}
+              justify="center"
+            >
+              <img
+                style={{ position: 'absolute', overflow: 'scroll' }}
+                alt="example"
+                src={img1}
+              />
+            </Col>
+          </Row>
+        </div>
+
+        <div className="site-card-border-less-wrapper">
+          <Card title="Game Play + Content" className="card" bordered={false}>
+            <p>1. STAR: Is Story Squad safe and appropriate for my children?</p>
```

Comment:
These look great, just a thought for a future design, making them be drop downs that bring out the answers would be an awesome way to show case them.